dotCMS/src/main/java/com/dotcms/mock/response/MockHttpStatusResponse.java

```diff
@@ -0,0 +1,36 @@
+package com.dotcms.mock.response;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+import java.io.File;
+import java.io.IOException;
+
+public class MockHttpStatusResponse extends HttpServletResponseWrapper implements MockResponse {
+
+    private int status = HttpServletResponse.SC_OK;
+
+    public MockHttpStatusResponse(HttpServletResponse response) {
+        super(response);
+    }
+
+    @Override
+    public HttpServletResponse response() {
+        return this;
+    }
+
+    @Override
+    public void sendError(int status) throws IOException {
+        this.status = status;
+    }
+
+    @Override
+    public void setStatus(int status) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'status' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4885343)